src/main/java/TodoCommand.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * To do command that creates a to do task.
+ */
+public class TodoCommand extends Command {
+
+    private String command;
+
+    /**
+     * Constructor method.
+     *
+     * @param command
+     */
+    public TodoCommand(String command) {
+        this.command = command;
+    }
+
+    /**
+     * Execute method for To do command.
```

Comment:
First sentence of method should be "Executes" instead of Execute

Reply:
Must have missed this out, thanks!